gematria/datasets/bhive_importer.h

```diff
@@ -70,7 +70,9 @@ class BHiveImporter {
   // `base_address` as the address of the first instruction in the basic block.
   absl::StatusOr<BasicBlockWithThroughputProto> ParseBHiveCsvLine(
       std::string_view source_name, std::string_view line,
-      double throughput_scaling = 1.0, uint64_t base_address = 0);
+      size_t machine_code_hex_column_index = 0,
+      size_t throughput_column_index = 1, double throughput_scaling = 1.0,
```

Comment:
If we give the column indices almost always, I'd remove the defaults.

Maybe we should remove defaults also from the scaling and the base address, default values just complicate things, but that's out of scope of this PR.

Reply:
I've removed the default column indices in this patch. I'll probably write a follow up removing the throughput scaling and base addresses.